website/source/docs/commands/environment.html.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: "docs"
+page_title: "Environment"
+sidebar_current: "docs-commands-environment"
+description: |-
+  Consul's behavior can be modified by certain environment variables.
+---
+
+# Environment variables
+
+The Consul CLI will read the following environment variables to set
+behavioral defaults. These can be overridden in all cases using
+command-line arguments; see the 
+[Consul Commands documentation](https://www.consul.io/docs/commands/index.html)
+for details.
+
+The following table describes these variables:
```

Comment:
A table isn't the best structure for this. It severely limits our ability to add code blocks and expand over time. Instead, I would take a page from Vagrant's book here and put each envvar in monofont as an h2 (`##`) with a description and examples. I also agree with @sean- that we should have lots of examples. Here is what I'm thinking:

```md
## `CONSUL_HTTP_ADDR`

This is the HTTP API address to the Consul agent. This should always point to the _local_
Consul agent, not a remote Consul server. This may be specified as a URI:

    CONSUL_HTTP_ADDR=127.0.0.1:8500

or via a UNIX socket:

    CONSUL_HTTP_ADDR=unix://var/run/consul.sock
```

<img width="688" alt="screen shot 2016-12-06 at 10 06 29 am" src="https://cloud.githubusercontent.com/assets/408570/20937409/b453a6da-bb9b-11e6-8c7b-24855062008b.png">